$NetBSD: patch-aa,v 1.1 2007/11/23 14:34:34 bjs Exp $

libxkbui does not depend on libXt (as per Matthieu Herrb).

--- configure.ac.orig	2006-04-26 15:59:50.000000000 -0400
+++ configure.ac
@@ -34,7 +34,7 @@ AC_PROG_CC
 AC_PROG_LIBTOOL
 
 # Check for dependencies
-PKG_CHECK_MODULES(XKBUI, x11 xt xkbfile)
+PKG_CHECK_MODULES(XKBUI, x11 xkbfile)
 AC_SEARCH_LIBS(hypot, [m])
 
 AC_SUBST(XKBUI_CFLAGS)
